installation/windows/scaffoldings.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Telerik UI for .NET MAUI Scaffoldings
+page_title: Telerik UI for .NET MAUI Scaffoldings - Visual Studio Scaffoldings
+description: Learn how to add predefined screens in your application using via Telerik .NET MAUI Visual Studio Scaffoldings.
+slug: maui-vs-scaffoldings
+position: 10
+tags: maui, dotnet maui, item templates, scaffoldings, predefined pages, screens, visual studio
+---
+
+# .NET MAUI Scaffoldings for Visual Studio
+
+The Telerik UI for .NET MAUI Visual Studio extension provides scaffoldings for increased developer productivity. The scaffoldings allow you to quickly add predefined pages with controls to your application and define the parameters of the controls through an UI.
+
+## Get the Extension
+
+To use the Telerik UI for .NET MAUI Scaffoldings, install the `Progress Telerik UI for .NET MAUI Extension` extension for Visual Studio. You can get the extension from:
+
+* <a href="https://marketplace.visualstudio.com/items?itemName=TelerikInc.ProgressTelerikMAUIExtensions" target="_blank">The Visual Studio Marketplace.</a>
+
+* The Extensions tab in Visual Studio  - search for `Progress Telerik UI for .NET MAUI Extension`, select the extension, and then click `Install`.
+
+* The Telerik UI for .NET MAUI [MSI installation]({%slug maui-getting-started%}). For more information, go to the [getting started with the Telerik VS extensions]({%slug visualstudio-extensions%}) topic.
+
+## Add The Scaffolding Page to Your App
+
+Here are the steps you need to follow to add the desired screen to your application:
+
+1. Open the project in Visual Studio.
+1. Right-click on the name of the project.
+1. From the popup menu select `Telerik UI for .NET MAUI` -> `Add new Scaffolded Item`.
+![.NET MAUI Scaffolding dialog](images/scaffolding-vs-dialog.png)
+
+1. From the UI form, select the desired page and set the available parameters.
+![.NET MAUI Scaffolding options](images/scaffolding-vs-options.png)
+
+1. The page will be added to the `Pages` folder of the application. if one does not exist, it will be created. Every page has a `ViewModel`, so the `ViewModel` is added to the `ViewModels` folder. if one does not exist, it will be created. The image shows the `Products` screen after adding to the project.
```

Comment:
```suggestion
As a result, the VS code extension does the following:
* Adds the page to the `Pages` folder of the application. If the folder does not exist, it will be created. 
* Adds a `ViewModel` for the created page to the `ViewModels` folder. If the folder does not exist, it will be created.

The next image shows the file structure created for the `Products` screen:
```